docs/helm-chart/index.rst

```diff
@@ -90,6 +90,10 @@ To install this chart using Helm 3, run the following commands:
     helm repo add apache-airflow https://airflow.apache.org
     helm upgrade --install airflow apache-airflow/airflow --namespace airflow --create-namespace
 
+**Explanation**:
+`helm repo add apache-airflow https://airflow.apache.org`
+- `helm repo add` is a command to add a chart repository
+
```

Comment:
Hey @AgnesNM thanks for your contribution.

I think it might be better rework the existing instructions for clarity rather than adding an explanation block.

So for example, replace this:

```
To install this chart using Helm 3, run the following commands:
.. code-block:: bash
    helm repo add apache-airflow https://airflow.apache.org
    helm upgrade --install airflow apache-airflow/airflow --namespace airflow --create-namespace
```

With 

```
To install this chart using Helm 3, first add the helm chart repository:

.. code-block:: bash
    helm repo add apache-airflow https://airflow.apache.org

Next, install the helm chart:

.. code-block:: bash
    helm upgrade --install airflow apache-airflow/airflow --namespace airflow --create-namespace
```

That way, we get the description saying what these commands actually do.  Better continuity compared with explaining in a diff section.